yaksh/templates/yaksh/courses.html

```diff
@@ -180,14 +230,146 @@
 </div>
 {% endif %}
 </div>
+<!-- *************************Show only Active Courses********************************* -->
+
+{% if type == "courses" %}
+<div id="show_only_active" style="display: none;">
+{% if not courses %}
+    <center><h4> No new Courses created </h4></center>
+{% else %}
+<div class="col-md-offset-2 main">
+    <center><h3> Course(s) Created</h3></center>
+    <table id="show_only_active" class="table table-bordered">
+    <tr>
+    <th>Courses</th>
```

Comment:
If there are courses but are not active, then we see this table with module and courses as th tags. Fix this